src/Entity/TaskImage.php

```diff
@@ -22,10 +36,11 @@ class TaskImage
     private $task;
 
     /**
-     * @Vich\UploadableField(mapping="task_image", fileNameProperty="imageName")
      * @var File
      */
-    #[Assert\File(maxSize: '5M', mimeTypes: ['image/jpg', 'image/jpeg', 'image/png'])]
+    #[Vich\UploadableField(mapping: 'task_image', fileNameProperty: 'imageName')]
+    // FIXME Make validation work again
```

Comment:
Are you planning to fix it here or in another issue?

Reply:
The issue is that `maxSize` thing doesn't work